sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: WWW Site Hits V2
+description: |-
+  GA4 normalized events for www.mozilla.org; events triggered at the same time are given the same hit number.
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: bqetl_google_analytics_derived_ga4
+bigquery:
+  time_partitioning:
+    type: day
+    field: date
```

Comment:
```suggestion
    field: event_date
```

Reply:
I unfortunately have to match the column names from the V1 table